README.md

```diff
@@ -59,10 +59,56 @@ credentials:
       aws_secret_access_key=MYKEYCONTENTS
 ```
 
+### Azure
+```yaml
+configuration:
+  provider: azure
+
+  backupStorageLocation:
+    name: default
+    provider: azure
+    # Blob container bucket
+    bucket: testing
+    config:
+      resourceGroup: r55p6
+      # Storage Account Name
+      storageAccount: testingveleroapp
+      bucket: testing
+
+initContainers:
+- name: velero-plugin-for-microsoft-azure
+      image: velero/velero-plugin-for-microsoft-azure:v1.1.1
```

Comment:
Can we update aws one to `v1.1.0` (latest) plz

Reply:
not sure if i have time to test it but i can bump it 😄